store/tikv/coprocessor.go

```diff
@@ -211,7 +211,7 @@ func (r *copRanges) split(key []byte) (*copRanges, *copRanges) {
 // rangesPerTask limits the length of the ranges slice sent in one copTask.
 const rangesPerTask = 25000
 
-func buildCopTasks(bo *Backoffer, cache *RegionCache, ranges *copRanges, desc bool, streaming bool) ([]*copTask, error) {
+func buildCopTasks(bo *Backoffer, cache *RegionCache, ranges *copRanges, desc bool, streaming bool, storeType kv.StoreType) ([]*copTask, error) {
```

Comment:
there are 6 parameters on this function, can we shrink the last three parameter to `req *kv.Request`?